bindings/python/iota_sdk/types/block/metadata.py

```diff
@@ -138,6 +144,12 @@ class TransactionFailureReason(Enum):
     MissingStakingFeature = 18
     FailedToClaimStakingReward = 19
     FailedToClaimDelegationReward = 20
+    TransactionCapabilityNativeTokenBurningNotAllowed = 21
+    TransactionCapabilityManaBurningNotAllowed = 22
+    TransactionCapabilityAccountDestructionNotAllowed = 23
+    TransactionCapabilityAnchorDestructionNotAllowed = 24
+    TransactionCapabilityFoundryDestructionNotAllowed = 25
+    TransactionCapabilityNftDestructionNotAllowed = 26
```

Comment:
can we remove the prefix "TransactionCapability"? I think it should be clear from the context and the indexation (21...26) that those are related to transaction capabilities?

Reply:
I would prefer to keep them as explicit as possible.
Also not sure why the indexes would give any indication that it's about tx caps?

Reply:
because it's ordered by their corresponding index, not by name. So they will always be grouped like this.

Reply:
They won't be grouped when/if we add more later

Reply:
mmh, okay then